sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -53,3 +53,12 @@ def test_predict_proba():
         clf = estimator(**parameters).fit(samples, labels)
         assert_array_almost_equal(clf.predict_proba([[1., 1.]]),
                                   np.array([[0.5, 0.5]]))
+
+
+def test_hard_clamping():
+    samples = [[1.,1.],[1.,0.],[0.,1.]]
+    labels = [1.,0.,-1.]
+    for estimator, parameters in ESTIMATORS:
+        clf = estimator(**parameters).fit(samples, labels)
```

Comment:
In the interest of decoupling the tests and making them more readable, could you please let the ESTIMATORS use the default value for alpha an instead do the following:

```python
for estimator, parameters in ESTIMATORS:
    parameters = parameters.copy()
    parameters['alpha'] = 1.0  # force hard-clamping
    clf = estimator(**parameters).fit(samples, labels)
    ...
```